README.md

```diff
@@ -0,0 +1,32 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+My first Project
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+👤 **Samiullah Bahadur**
+
+- GitHub: [@githubhandle](https://github.com/samiullahbahadur)
+- Twitter: [@twitterhandle](https://twitter.com/@Samiull88496331)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/samiullah-bahadur-a1b053149/)
```

Comment:
- [ ] You should change ID names. For example instead of [@githubhandle] you can write your id [@samiullahbahadur ]

Reply:
> * [ ]  You should change ID names. For example instead of [@githubhandle] you can write your id [@samiullahbahadur ]

- [ ] As I mentioned earlier, it's important to apply all the required changes by previous reviewers before asking for a new CR.